lib/bolt/plugin/module.rb

```diff
@@ -199,6 +199,7 @@ def extract_task_parameter_schema
       end
 
       def run_task(task, opts)
+        opts = opts.reject{ |key, _val| key.start_with?('_') }
```

Comment:
Purely out of curiosity is there a reason not to use `reject!` here?

Reply:
It's generally bad form to modify arguments passed into a function. If the caller wanted to pass `opts` to `run_task` and then use it for something else after, they would find it modified.